lib/ansible/modules/network/nxos/nxos_interface.py

```diff
@@ -277,70 +277,77 @@ def get_interface(intf, module):
+                    else:
+                        interface['fabric_forwarding_anycast_gateway'] = False
+
+                elif intf_type == 'loopback':
+                    key_map.update(base_key_map)
+                    key_map.pop('admin_state')
+                    key_map.update(loop_map)
+                    temp_dict = apply_key_map(key_map, interface_table)
+                    if not temp_dict.get('description'):
+                        temp_dict['description'] = "None"
+                    interface.update(temp_dict)
+
+                elif intf_type == 'management':
+                    key_map.update(base_key_map)
+                    temp_dict = apply_key_map(key_map, interface_table)
+                    interface.update(temp_dict)
+
+                elif intf_type == 'portchannel':
+                    key_map.update(base_key_map)
+                    key_map.update(mode_map)
+                    temp_dict = apply_key_map(key_map, interface_table)
+                    temp_dict = apply_value_map(mode_value_map, temp_dict)
+                    if not temp_dict.get('description'):
+                        temp_dict['description'] = "None"
+                    interface.update(temp_dict)
+
+                elif intf_type == 'nve':
+                    key_map.update(base_key_map)
+                    temp_dict = apply_key_map(key_map, interface_table)
+                    if not temp_dict.get('description'):
+                        temp_dict['description'] = "None"
+                    interface.update(temp_dict)
             else:
-                interface['fabric_forwarding_anycast_gateway'] = False
-
-        elif intf_type == 'loopback':
-            key_map.update(base_key_map)
-            key_map.pop('admin_state')
-            key_map.update(loop_map)
-            temp_dict = apply_key_map(key_map, interface_table)
-            if not temp_dict.get('description'):
-                temp_dict['description'] = "None"
-            interface.update(temp_dict)
-
-        elif intf_type == 'management':
-            key_map.update(base_key_map)
-            temp_dict = apply_key_map(key_map, interface_table)
-            interface.update(temp_dict)
-
-        elif intf_type == 'portchannel':
-            key_map.update(base_key_map)
-            key_map.update(mode_map)
-            temp_dict = apply_key_map(key_map, interface_table)
-            temp_dict = apply_value_map(mode_value_map, temp_dict)
-            if not temp_dict.get('description'):
-                temp_dict['description'] = "None"
-            interface.update(temp_dict)
-
-        elif intf_type == 'nve':
-            key_map.update(base_key_map)
-            temp_dict = apply_key_map(key_map, interface_table)
-            if not temp_dict.get('description'):
-                temp_dict['description'] = "None"
-            interface.update(temp_dict)
+                interface_table = ""
```

Comment:
Why add this? Unless I'm missing something, `interface_table` isn't used at this point

Reply:
I have no idea why I did that. Thanks for the catch. updated the fix now.